internal/cli/attach.go

```
@@ -0,0 +1,58 @@
+			if len(procs) != 1 {
+				return errors.Newf("expected 1 process, found %d", len(procs))
+			}
+			proc := procs[0]
+
+			conn, err := net.Dial("unix", filepath.Join(core.DirHome, proc.ID.String()+".sock"))
+			if err != nil {
+				return errors.Wrap(err, "connect to proc socket")
+			}
+
+			go func() {
+				_, err := io.Copy(os.Stdout, conn)
```

golangci-lint (1.62.2)

shadow: declaration of "err" shadows declaration at line 41

(govet)